utilities/__init__.py

```diff
@@ -0,0 +1,4 @@
+import utilities_setup
+from utilities_functions import save__temp_field_chamber_res_data
+from utilities_functions import scan_field_no_keithley

```

Comment:
You can combine this line and the next line into one.

Reply:
```suggestion

from scan_functions import scan_field_no_keithley, scan_field_with_keithley
```